packages/run-protocol/src/runStake/runStake.js

```diff
@@ -12,67 +12,65 @@ const { values } = Object;
 /**
  * Provide loans on the basis of staked assets that earn rewards.
  *
- * In addition to brands and issuers for `Staked`, `RUN`, and attestation,
- * terms of the contract include a periodic `InterestRate`
- * plus a `LoanFee` proportional to the amount borrowed, as well as
- * a `MintingRatio` of funds to (mint and) loan per unit of staked asset.
- * These terms are subject to change by the `Electorate`
- * and `electionManager` terms.
+ * In addition to brands and issuers for `Staked`, `RUN`, and attestation, terms
+ * of the contract include a periodic `InterestRate` plus a `LoanFee`
+ * proportional to the amount borrowed, as well as a `MintingRatio` of funds to
+ * (mint and) loan per unit of staked asset. These terms are subject to change
+ * by the `Electorate` and `electionManager` terms.
  *
  * @typedef {{
- *   MintingRatio: ParamRecord<'ratio'>,
- *   InterestRate: ParamRecord<'ratio'>,
- *   LoanFee: ParamRecord<'ratio'>,
+ *   MintingRatio: ParamRecord<'ratio'>;
+ *   InterestRate: ParamRecord<'ratio'>;
+ *   LoanFee: ParamRecord<'ratio'>;
  * }} RunStakeParams
+ *   As in vaultFactory, `timerService` provides the periodic signal to charge
+ *   interest according to `chargingPeriod` and `recordingPeriod`.
  *
- * As in vaultFactory, `timerService` provides the periodic signal to
- * charge interest according to `chargingPeriod` and `recordingPeriod`.
- *
- * @typedef { GovernanceTerms<RunStakeParams> & {
- *   timerService: TimerService,
- *   chargingPeriod: bigint,
- *   recordingPeriod: bigint,
- *   lienAttestationName?: string,
+ * @typedef {GovernanceTerms<RunStakeParams> & {
+ *   timerService: TimerService;
+ *   chargingPeriod: bigint;
+ *   recordingPeriod: bigint;
+ *   lienAttestationName?: string;
  * }} RunStakeTerms
- *
- * The public facet provides access to invitations to get a loan
- * or to return an attestation in order to release a lien on staked assets.
+ *   The public facet provides access to invitations to get a loan or to return an
+ *   attestation in order to release a lien on staked assets.
  *
  * @typedef {{
- *   makeLoanInvitation: () => Promise<Invitation>,
- *   makeReturnAttInvitation: () => Promise<Invitation>,
+ *   makeLoanInvitation: () => Promise<Invitation>;
+ *   makeReturnAttInvitation: () => Promise<Invitation>;
  * }} RunStakePublic
+ *   To take out a loan, get an `AttestationMaker` for your address from the
```

Comment:
@dckc the formatter reads the paragraph following a typedef as being about that typedef and so indents it.

I think this will be fixed by `https://github.com/hosseinmd/prettier-plugin-jsdoc/issues/156`